.rubocop.yml

```diff
@@ -0,0 +1,14 @@
+Style/FrozenStringLiteralComment:
+  Enabled: false
+
+Metrics/LineLength:
+  Max: 120
+
+Style/Documentation:
+  Enabled: false
+
+Metrics/MethodLength:
+  Max: 30
```

Comment:
It is possible to change method length , but better to do methods small, make Max: 10 and add changes to method